ci-operator/step-registry/observers/oink/observers-oink-commands.sh

```
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function cleanup() {
+  local rc=$?
+  printf "%s: Stop recording \n" "$(date --utc --iso=s)"
+  echo "killing resource watch"
+  readarray -t CHILDREN < <(jobs -p)
+  if [ ${`#CHILDREN`[@]} -gt 0 ]; then
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)